README.md

```diff
@@ -148,6 +148,7 @@ For local development and familiarizing yourself with Storm/Mesos, please see th
 * `mesos.master.failover.timeout.secs`: Framework failover timeout in second. Defaults to "24*7*3600".
 * `mesos.allowed.hosts`: Allowed hosts to run topology, which takes hostname list as a white list.
 * `mesos.disallowed.hosts`: Disallowed hosts to run topology, which takes hostname list as a back list.
+* `mesos.constraints`: Constrain slave nodes which bootstrap supervisor based on mesos-attributes.  
```

Comment:
This needs more info for it to be useful without someone reading the related code.  e.g., it took us time to realize that if you had a list of constraints, then all of those text attributes must both exist in the offer's attributes **and** their values must be identical.  i.e., if you had an attribute in the "constraint" that didn't exist in the Offer it would be rejected.  We just need to be very explicit with our documentation around this.

Instead of increasing the length of this comment, let's put in a section in the README about configuring this feature, both within storm-mesos and for the mesos-agent.  We can link to the Mesos docs, and put in an example of configuring both mesos-agent and storm-mesos.